APE0.rst

```diff
@@ -0,0 +1,360 @@
+work to:
+
+* Maintain the quality and stability of the Astropy ecosystem of packages for
+  the community,
+* Facilitate the development of resources that improve accessibility and
+  understanding of the Astropy ecosystem,
+* Make contributing as accessible, inclusive, and sustainable as possible,
+* Oversee the financial and legal status of the project,
+* Lead and oversee the decision-making process for APEs and other community
+  decisions,
+* Seek consensus among the Astropy community, particularly before acting in a
+  formal capacity,
+* Act as the decision-making body where all other methods for achieving
+  consensus have failed,
+* Operate openly and provide transparency and opportunity for community input
+  into its activities.
+
+Powers
+^^^^^^
+The Coordination Committee has broad authority to make decisions about the
+Project. For example, they can:
+
+* Accept or reject APEs and updates to APEs (except this one - see below);
+* Enforce or update the Project's code of conduct
+* Manage any project assets (including financial and physical assets)
+* Delegate parts of their authority to other subcommittees or processes
+
+However, they cannot modify the Governance of the Project (i.e., this document),
+except during the initial adoption of this Charter as specified herein.
+
+The Coordination Committee has broad authority, which it seeks to exercise as
+rarely as possible by working towards consensus. The Coordination Committee
+should look for ways to use these powers as little as possible, striving to
+involve the community whenever possible for decisions, and delegating
+responsibilities.
+
+When delegating its power by creating roles or making appointments the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its powers, the Coordination Committee must first attempt to reach a
+consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active s) at least one month before the election begins.
```

Comment:
"all active s"?

Reply:
yes, that was definitely a typo, thanks @embray and @pllim !